2020-04-24 13:48:12,858 ERROR 8688 nodejs.ClusterClientNoResponseError: client no response in 5146243ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\reactBlog\service\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 8688
hostname: AUUX0RJNSI27X9S

2020-04-25 10:32:46,252 ERROR 18956 [-/127.0.0.1/-/7ms POST /admin/checkLogin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'liuskr' in 'where clause'
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName=liuskr AND password=123456
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'liuskr' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName=liuskr AND password=123456"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 18956
hostname: AUUX0RJNSI27X9S

2020-04-25 10:36:43,394 ERROR 8752 [-/127.0.0.1/-/15ms POST /admin/checkOpenId] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'asd' in 'where clause'
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName=asd  AND password=123456
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'asd' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName=asd  AND password=123456"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 8752
hostname: AUUX0RJNSI27X9S

2020-04-25 10:37:17,887 ERROR 1896 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 1896
hostname: AUUX0RJNSI27X9S

2020-04-25 10:46:51,325 ERROR 11636 [-/127.0.0.1/-/16ms POST /admin/checkOpenId] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'asd' in 'where clause'
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName=asd  AND password=12123
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'asd' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName=asd  AND password=12123"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 11636
hostname: AUUX0RJNSI27X9S

2020-04-25 10:48:04,317 ERROR 18084 [-/127.0.0.1/-/18ms POST /admin/checkOpenId] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'asd' in 'where clause'
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName=asd  AND password=123456
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'asd' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName=asd  AND password=123456"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 18084
hostname: AUUX0RJNSI27X9S

2020-04-25 10:50:10,710 ERROR 19152 [-/127.0.0.1/-/16ms POST /admin/checkOpenId] nodejs.ReferenceError: openId is not defined
    at MainController.checkLogin (E:\reactBlog\service\app\controller\admin\main.js:19:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async E:\reactBlog\service\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (E:\reactBlog\service\node_modules\egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (E:\reactBlog\service\node_modules\egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (E:\reactBlog\service\node_modules\egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (E:\reactBlog\service\node_modules\egg-security\lib\middlewares\noopen.js:8:5)
    at async session (E:\reactBlog\service\node_modules\koa-session\index.js:41:7)
    at async bodyParser (E:\reactBlog\service\node_modules\koa-bodyparser\index.js:95:5)
    at async E:\reactBlog\service\node_modules\koa-static-cache\index.js:51:63
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 19152
hostname: AUUX0RJNSI27X9S

2020-04-25 10:52:57,598 ERROR 17616 [-/127.0.0.1/-/17ms POST /admin/checkOpenId] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'liuskr' in 'where clause'
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName=liuskr AND password=123456
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'liuskr' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName=liuskr AND password=123456"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 17616
hostname: AUUX0RJNSI27X9S

2020-04-25 11:22:03,854 ERROR 18088 [-/127.0.0.1/-/16ms POST /admin/checkOpenId] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' AND password='qqq'' at line 1
    at Query.Sequence._packetToError (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (E:\reactBlog\service\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (E:\reactBlog\service\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    --------------------
    at Pool.query (E:\reactBlog\service\node_modules\mysql\lib\Pool.js:199:23)
    at E:\reactBlog\service\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (E:\reactBlog\service\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (E:\reactBlog\service\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (E:\reactBlog\service\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (E:\reactBlog\service\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (E:\reactBlog\service\node_modules\co\index.js:65:19)
    at E:\reactBlog\service\node_modules\co\index.js:54:5
    sql: SELECT userName FROM admin_user WHERE userName='' or 1=1' AND password='qqq'
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' AND password='qqq'' at line 1"
sqlState: "42000"
index: 0
sql: "SELECT userName FROM admin_user WHERE userName='' or 1=1' AND password='qqq'"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 18088
hostname: AUUX0RJNSI27X9S

2020-04-25 13:35:57,864 ERROR 18160 nodejs.ClusterClientNoResponseError: client no response in 4312894ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\reactBlog\service\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 18160
hostname: AUUX0RJNSI27X9S

2020-04-26 08:59:34,784 ERROR 18160 nodejs.ClusterClientNoResponseError: client no response in 51666387ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\reactBlog\service\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 18160
hostname: AUUX0RJNSI27X9S

